appserver/tests/payara-samples/samples/realm-identity-stores/src/main/webapp/WEB-INF/beans.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`beans.xml` should not be necessary (with scope annotations placed on relevant classes).